app/scripts/views/complete_reset_password.js

```diff
@@ -28,6 +30,19 @@ define(function (require, exports, module) {
     initialize (options) {
       options = options || {};
 
+      this.flow = new Flow({
```

Comment:
I was just commenting to @philbooth that I don't see an equivalent section of code in complete_sign_up, is that expected? If we do decide we need it over there too, can this be extracted into a common location and shared?